src/DatePicker/datePicker.scss

```diff
@@ -1,5 +1,5 @@
 @import '../utils/theme/variables.scss';
-@import 'react-dates/lib/css/_datepicker';
+@import '~react-dates/lib/css/_datepicker.css';
```

Comment:
related?

Reply:
I had to rebase on https://github.com/buildo/react-components/pull/1474 (not yet merged) to make it compile.